superset-frontend/cypress-base/cypress/e2e/dashboard/horizontalFilterBar.test.ts

```diff
@@ -81,7 +81,7 @@ describe('Horizontal FilterBar', () => {
     setFilterBarOrientation('vertical');
   });
 
-  it('should show all default actions in horizontal mode', () => {
+  it.skip('should show all default actions in horizontal mode', () => {
```

Comment:
instead of doing all the skips here and below, you can just do `it.only` and it'll only run that test above. You can probably put a note after that test to list out why you're skipping all these other tests